dxr/build.py

```diff
@@ -523,6 +522,11 @@ def docs():
                     total['annotations'] = annotations_for_this_line
                 yield es.index_op(total)
 
+                # Because needles_by_line holds a reference, total is not
```

Comment:
![shallowcpy](https://cloud.githubusercontent.com/assets/2406051/15757769/0f5c103a-28bc-11e6-908f-6f907b7150db.png)
.copy()
![mandreel_better_w_sorted](https://cloud.githubusercontent.com/assets/2406051/15757803/40914940-28bc-11e6-8f0b-2f89958d0ccc.png)
.clear()
Looks like a small difference, I can switch over to the copying.